lesson02/assignment/index.html

```diff
@@ -4,11 +4,32 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta http-equiv="X-UA-Compatible" content="ie=edge" />
         <link rel="stylesheet" href="css/style.css" />
+        <script src="https://kit.fontawesome.com/5b1c172637.js" crossorigin="anonymous"></script>
         <title>Assignment 02</title>
     </head>
     <body>
         <main class="main">
-            <h1>title</h1>
+          <div class="col">
+            <div class="card">
+              <div class="section-1">
+                <img src="img/unsplash-headshot.jpg" alt="profile-pic">
+                <h3>Steve Smith</h3>
+                <p>Project Manager</p>
+              </div>
+              <div class="section-2">
+                <p>Company: <span>Front End Dev Co</span></p>
+                <p>Experience: <span>3 years</span></p>
+                <p>School: <span>UW</span></p>
+                <p>Major: <span>Marketing</span></p>
+                <p>Email: <span>steve@fedc.com</span></p>
+                <p style="font-size: 1em;color: #007bb6;">
```

Comment:
Modern HTML never includes inline styles like this (except when coding for emails). All styling should be in CSS or Sass.